src/main/java/org/apache/maven/plugins/javadoc/AbstractJavadocMojo.java

```diff
@@ -6081,7 +6081,25 @@ && isJavadocVMInitError( output ) )
                 msg.append( exitCode );
                 if ( StringUtils.isNotEmpty( err.getOutput() ) )
                 {
-                    msg.append( " - " ).append( err.getOutput() );
+                    // parse stderr, log informational output, add all other to exception message
+                    List<String> nonInfoLines = new ArrayList<>();
+                    for ( String str : err.getOutput().split( "\\R" ) )
```

Comment:
I assume that `getOutput()` contains everything as a string?

Reply:
correct

The signature: `String StringStreamConsumer.getOutput()`